docs/targets/logz.md

```diff
@@ -0,0 +1,47 @@
+# Event Target for Logz.io 
+This event target integrates with Logz.io, using received CloudEvent's to ship messages to [Logz.io](https://logz.io/)
+
+## Prerequisites
+* Logz.io account.
```

Comment:
We do not need the ending period since this is not a complete sentence. 

What do you think of adding a link to the create account page for log.io?